runtime/themes/nord_light.toml

```
@@ -58,6 +58,10 @@
 "markup.link.text" = {fg="nord12"}
 "markup.quote" = {fg="nord3", modifiers=["italic"]}
 
+"diff.plus" = {fg = "nord14"}
+"diff.delta" = {fg = "nord11"}
+"diff.minus" = {fg = "nord15"}
```

```suggestion
"diff.plus" = {fg = "nord14"}
"diff.delta" = {fg = "nord13"}
"diff.minus" = {fg = "nord11"}
```

This is per descriptions in https://www.nordtheme.com/docs/colors-and-palettes